da-test/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "da-test"
+version = "0.1.0"
+edition = "2021"
+
+
+[dependencies]
```

Comment:
Declare the deps in workspace Cargo.toml and import them from there.
It will avoid us discrepancies when we bump our deps